src/man/sssd-passkey.5.xml

```diff
@@ -0,0 +1,120 @@
+    </refsect1>
+
+    <refsect1 id='configuration-options'>
+        <title>CONFIGURATION OPTIONS</title>
+        <para>
+            <variablelist>
+                <varlistentry>
+                    <term>user_verification (boolean)</term>
+                    <listitem>
+                        <para>
+                            Enable or disable the user verification (i.e. PIN,
+                            fingerprint) during authentication.
+                        </para>
+                        <para>
+                            Three different actors come into play when deciding
+                            whether to request user verification: LDAP server,
+                            <citerefentry><refentrytitle>sssd.conf
+                            </refentrytitle><manvolnum>5</manvolnum></citerefentry>
+                            option and the passkey device itself.
+                        </para>
+                        <para>
+                            For the IPA provider the server side settings have
+                            higher priority than <filename>sssd.conf
+                            </filename>. For every other provider the settings
+                            from <filename>sssd.conf</filename> have higher
+                            priority.
+                        </para>
+                        <para>
+                            In the case of the <filename>sssd.conf</filename>
+                            and the passkey device options the situation is
+                            explained in the following table:
+                        </para>
+                        <informaltable frame='all'>
+                        <tgroup cols='3'>
+                        <colspec colname='c1' align='center'/>
+                        <colspec colname='c2' align='center'/>
+                        <colspec colname='c3' align='center'/>
+
+                        <thead>
+                        <row><entry>IPA server/sssd.conf</entry><entry>Device</entry>
+                            <entry>Result</entry></row>
+                        </thead>
+                        <tbody>
+                        <row>
+                            <entry>True</entry>
+                            <entry>User verification is configured</entry>
+                            <entry>User verification is requested</entry>
+                        </row>
+                        <row>
+                            <entry>True</entry>
+                            <entry>User verification is not configured</entry>
+                            <entry><para>
+                            User verification is requested; however, the
+                            authentication is expected to fail if the device is
+                            not replaced with a device where authentication is
+                            not configured during the authentication process.
```

Comment:
Shouldn't this be " ... replaced with a device where user verification _is_ configured ..."?

Reply:
Yes, you are right

Reply:
I still think it would be better to say "... where user verification is configured ..." instead of "... where authentication is configured ..." to make a more clear distinction between the authentication the user is doing against the system to log in and the means needed to prove that the user is the owner of the passkey device.

Reply:
You are right, I think I mistyped there